packages/instant-apps-components/src/assets/t9n/instant-apps-language-translator/resources.json

```diff
@@ -0,0 +1,102 @@
+{
+  "close": "Close",
+  "collapseAll": "Collapse all",
+  "expandAll": "Expand all",
+  "goToSetting": "Go to setting",
+  "header": "Language translator",
+  "headerTip": "Language translator tip",
+  "languages": {
+    "ar": "Arabic",
+    "bg": "Bulgarian",
+    "bs": "Bosnian",
+    "ca": "Catalan",
+    "cs": "Czech",
+    "da": "Danish",
+    "de": "German",
+    "el": "Greek",
+    "en": "English",
+    "es": "Spanish",
+    "et": "Estonian",
+    "fi": "Finnish",
+    "fr": "French",
+    "he": "Hebrew",
+    "hr": "Croatian",
+    "hu": "Hungarian",
+    "id": "Indonesian",
+    "it": "Italian",
+    "ja": "Japanese",
+    "ko": "Korean",
+    "lt": "Lithuanian",
+    "lv": "Latvian",
+    "nb": "Norwegian",
+    "nl": "Dutch",
+    "pl": "Polish",
+    "pt-BR": "Portuguese (Brazil)",
+    "pt-PT": "Portuguese (Portugal)",
+    "ro": "Romanian",
+    "ru": "Russian",
+    "sk": "Slovak",
+    "sl": "Slovenian",
+    "sr": "Serbian",
+    "sv": "Swedish",
+    "th": "Thai",
+    "tr": "Turkish",
+    "uk": "Ukrainian",
+    "vi": "Vietnamese",
+    "zh-CN": "Simplified Chinese",
+    "zh-HK": "Traditional Chinese",
```

Comment:
We might want to add Hong Kong and Taiwan after traditional Chinese for zh-CN and zh-HK.

Reply:
Done